doc/developer-guides/hld/hv-interrupt.rst

```diff
@@ -63,7 +63,9 @@ to support this. The ACRN hypervisor also initializes all the interrupt
 related modules like IDT, PIC, IOAPIC, and LAPIC.
 
 HV does not own any host devices (except UART). All devices are by
-default assigned to the Service VM. Any interrupts received by Guest VM (Service VM or User VM) device drivers are virtual interrupts injected by HV (via vLAPIC).
+default assigned to the Service VM. Any interrupts received by Guest VM
```

Comment:
Using "Guest VM" here makes me feel we're only talking about User VMs, how about we don't specify it and more generically say "received by Virtual Machines (Service VM and User VM)"